lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Form\Processor;
+
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Content\ContentStruct;
+use eZ\Publish\Core\MVC\Symfony\Routing\UrlAliasRouter;
+use EzSystems\RepositoryForms\Event\FormActionEvent;
+use EzSystems\RepositoryForms\Event\RepositoryFormEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+class ContentFormProcessor implements EventSubscriberInterface
```

Comment:
I've added a phpdoc for the class itself. The methods do exactly what they say, and the parameters (the event) are self-explanatory, imho. When I have more to add, I will :-)